src/runtime/pkg/resourcecontrol/controller.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 Apple Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package resourcecontrol
+
+import (
+	v1 "github.com/containerd/cgroups/stats/v1"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	controllerLogger = logrus.WithField("source", "virtcontainers/pkg/cgroups")
```

Comment:
Should this be instead?:

```suggestion
	controllerLogger = logrus.WithField("source", "virtcontainers/pkg/resourcecontrol")
```

Reply:
You're right, fixed now. Thanks.